### ISSUE TITLE
32 dim pretrained model download

### ISSUE THREAD

**Issue by TobiasFischerP4D** [zgojcic/3DSmoothNet#28]
Thank you for publishing the code used in your work :)

I would like to use the model, specifically the 32 dimensional descriptor. The paper specifically investigates this dimensionality, also it should run a little faster and requires only 1/4th of the memory to store the features.

The paper states:
`Code, data and pre-trained models are available online at [...]`

However there is only a single pretrained model linked:[
https://share.phys.ethz.ch/~gsg/3DSmoothNet/models/128_dim/3DSmoothNet_model_128_dim.rar](
https://share.phys.ethz.ch/~gsg/3DSmoothNet/models/128_dim/3DSmoothNet_model_128_dim.rar)

I tried to be smart and replaced the `128` with `32`, but that resulted in `404` :(

Could you provide the `16` and `32` dimensional pretrained models as well?

**Comment by zgojcic** [comment]
Hi, all the pretrained models, except the 128-dim, are provided directly in this repository in 3DSmoothNet/models/x_dim/. The 128-dim model is to big to upload it to github this is why we provided the link. If you want to use the 128_dim model you should download the file and extract it to /3DSmoothNet/models/128_dim/

**Comment by TobiasFischerP4D** [comment]
Oh well :see_no_evil: 
Thanks a lot!
